cmd/termination-handler/main.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"k8s.io/klog"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/termination"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/version"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func main() {
+	var printVersion bool
+	flag.BoolVar(&printVersion, "version", false, "print version and exit")
+
+	klog.InitFlags(nil)
+	pollInterval := flag.Duration("poll-interval", 5*time.Second, "interval at which termination notice endpoint should be checked")
```

Comment:
is this expecting nanoseconds? do we need that level? may be clarify expected units/format in the desc, e.g seconds?

Reply:
We definitely shouldn't be polling that quickly, I wonder if it's better to change this to `poll-interval-seconds` as an `int`, that might be a better way show users that this should be in the order of seconds rather than milliseconds, nanoseconds, minutes etc

WDYT?

Reply:
either way  works for me. I'm good with a duration may be just set an example in the flag desc e.g "5s"